src/defs/resourceServerRepresentation.ts

```diff
@@ -11,9 +11,15 @@ export enum PolicyEnforcementMode {
   DISABLED = 'DISABLED',
 }
 
+export enum DecisionStrategyOption {
+  UNANIMOUS = 'UNANIMOUS',
+  AFFIRMATIVE = 'AFFIRMATIVE',
```

Comment:
Unlike in Java, enums in TypeScript follow a camel case naming convention (see the TypeScript [documentation])(https://www.typescriptlang.org/docs/handbook/enums.html).
```suggestion
  Unanimous = 'UNANIMOUS',
  Affirmative = 'AFFIRMATIVE',
```

Reply:
I did it like that because it's used like that throughout the library

Reply:
okay, maybe we do want it more typescirptty... changed it to a type..